net-dns/libidn2/libidn2-2.0.4.recipe

```diff
@@ -0,0 +1,72 @@
+SUMMARY="Internationalized Domain Names (IDN) implementation"
+DESCRIPTION="Libidn2 is a free software implementation of IDNA2008, Punycode and TR46. Its \
+purpose is to encode and decode internationalized domain names."
+HOMEPAGE="https://gilab.com/libidn/libidn2"
+COPYRIGHT="2011-2016 Simon Josefsson"
+LICENSE="GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="http://ftp.gnu.org/gnu/libidn/libidn2-$portVersion.tar.gz"
+CHECKSUM_SHA256="644b6b03b285fb0ace02d241d59483d98bc462729d8bb3608d5cad5532f3d2f0"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	libidn2$secondaryArchSuffix = $portVersion
+	lib:libidn2$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:gettext$secondaryArchSuffix
+	lib:texinfo$secondaryArchSuffix
```

Comment:
There's [no](https://github.com/haikuports/haikuports/blob/master/sys-devel/gettext/gettext-0.19.8.1.recipe#L37) `lib:gettext` [nor](https://github.com/haikuports/haikuports/blob/master/sys-apps/texinfo/texinfo-6.5.recipe#L14) `lib:texinfo`

Reply:
I thought that those were necessary tools to build libidn2 - are they not needed at all?

Reply:
What I mean is no package actually provides them. See the link I embed in the original comment